packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js

```diff
@@ -1,7 +1,30 @@
 export const BANDWIDTH_OUT = 'bandwidth_instance_out.consumption';
 export const EXCLUDE_FLAVOR_CATEGORIES = ['baremetal'];
 
+export const INSTANCE_MODES_ENUM = [
+  { mode: 'public_mode' },
+  { mode: 'private_mode' },
+];
+
+export const AVAILABLE_SUBNET = [
+  '192.168.0.1/24',
+  '10.0.0.1/24',
+  '172.16.0.1/24',
+];
```

Comment:
**question:** Is it okay to maintain these subnets? Shouldn't we get it from API?

Reply:
yeah, we don't get this subnet values from API. As per the discussion with Kuba during gateway development, this value should be hardcoded.